app/src/main/java/org/covidwatch/android/ui/menu/MenuFragment.kt

```diff
@@ -63,4 +73,23 @@ class MenuFragment : Fragment(R.layout.fragment_menu) {
         findNavController().navigate(R.id.homeFragment)
     }
 
+    //Set red menu icon visible if maximum risk score > safe risk score
+    private fun updatePossibleExposureMenuItem(expSummary: CovidExposureSummary) {
+        val menuListAdapter: MenuAdapter = saveMenuList.adapter as MenuAdapter
```

Comment:
Instead of keeping the reference to the list keep the adapter and use it directly instead of casting every time.